sale_financial_risk_info/tests/test_sale_financial_risk_info.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
+
+from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+
+
+class TestSaleFinancialRiskInfo(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
+        cls.Partner = cls.env["res.partner"]
+        cls.Product = cls.env["product.product"]
+
+        cls.product_1 = cls.Product.create(
+            {"name": "Product test 1", "list_price": 1000.00}
+        )
+        cls.partner_1 = cls.Partner.create(
+            {
+                "name": "partner test rebate 1",
+                "ref": "TST-001",
+                "risk_invoice_draft_include": True,
+            }
+        )
+        cls.invoice_partner_1 = cls.create_invoice(cls.partner_1)
+
+    # Create some invoices for partner
+    @classmethod
+    def create_invoice(cls, partner):
+        simple_invoice = cls.env["account.move"].create(
+            {
+                "move_type": "out_invoice",
+                "partner_id": partner.id,
+                "ref": "Test Customer Invoice",
+                "invoice_line_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "product_id": cls.product_1.id,
+                            "quantity": 1,
+                        },
+                    ),
```

Comment:
Could you use the [Command](https://github.com/odoo/odoo/blob/370de95190cf07f17281f0ed6365b6806663cac2/odoo/fields.py#L4227-L4256) class? It is more readable.

```suggestion
                    Command.create(
                        {
                            "product_id": cls.product_1.id,
                            "quantity": 1,
                        },
                    ),
```